Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityEventHandlers.cs

```diff
@@ -0,0 +1,423 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using static UnityEngine.EventSystems.ExecuteEvents;
+
+namespace Microsoft.MixedReality.Toolkit.Input.Utilities
```

Comment:
It looks like you don't use this anywhere, did you mean to add this file? Perhaps it was added by accident?